multi-storage-transaction-sample/README.md

```diff
@@ -123,35 +125,43 @@ scalar.db.multi_storage.namespace_mapping=customer:mysql,order:cassandra,coordin
-The tables in the `coordinator` namespace are created automatically and used in the ScalarDB's transaction protocol called Consensus Commit. 
-And the `scalar.db.multi_storage.default_storage` property defines the default storage that’s used if a specified table doesn't have any table mapping.
-In this case, if a specified table doesn't have any table mapping, operations for the table are mapped to the `cassandra` storage.
+- `scalar.db.storage`: Specifying `multi-storage` is necessary to use Multi-storage Transactions in ScalarDB.
+- `scalar.db.multi_storage.storages`: Your storage names must be defined here.
+- `scalar.db.multi_storage.storages.cassandra.*`: These configurations are for the `cassandra` storage, which is one of the storage names defined in `scalar.db.multi_storage.storages`. You can configure all the `scalar.db.*` properties for the `cassandra` storage here.
+- `scalar.db.multi_storage.storages.mysql.*`: These configurations are for the `mysql` storage, which is one of the storage names defined in `scalar.db.multi_storage.storages`. You can configure all the `scalar.db.*` properties for the `mysql` storage here.
+- `scalar.db.multi_storage.namespace_mapping`: This configuration maps the namespaces to the storage. In this sample application, operations for `customer` namespace tables are mapped to the `mysql` storage and operations for `order` namespace tables are mapped to the `cassandra` storage. You can also define which storage is mapped for the `coordinator` namespace that is used in Consensus Commit transactions.
+- `scalar.db.multi_storage.default_storage`: This configuration sets the default storage that is used for operations on unmapped namespace tables.
+
+For details, please see [Configuration - Multi-storage Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/multi-storage-transactions.md#configuration).
 
-Please see below for the details of Multi-storage Transaction configurations:
-https://github.com/scalar-labs/scalardb/blob/master/docs/multi-storage-transactions.md
+## Setup Cassandra and MySQL
 
-## Set up
+To start Cassandra and MySQL, you need to run the following `docker-compose` command:
 
-You need to run the following `docker-compose` command:
 ```shell
 $ docker-compose up -d
 ```
 
-This command starts Cassandra and MySQL, and loads the schema.
-Please note that you need to wait around more than one minute for the containers to be fully started.
+Please note that starting the containers may take more than one minute.
 
-### Initial data
+## Load schema
+
+You then need to apply the schema with the following command.
+Please download the schema tool `scalardb-schema-loader-<version>.jar` that can be found in [releases](https://github.com/scalar-labs/scalardb/releases) of ScalarDB.
+
+```shell
+$ java -jar scalardb-schema-loader-<version>.jar --config database.properties --schema-file schema.json --coordinator
+```
```

Comment:
The most major change is here.

Previously, we loaded the schema using the docker-compose command. However, I've modified the process to download the schema loader jar and load the schema using that instead.